docs/developer-guide/release.md

```diff
@@ -74,7 +74,8 @@ Note- the date should be the date of the release in the format sh run.sh make GH
       - click edit
       - add the name of the new release and the content from the change log text file above. 
       - commit to master
-1. Commit release files
+   3. Click on "Publish release" (THIS IS NEW, the release, so far should have been in Draft state)
+1. Commit release files (<- Why is this necessary? This should be done in the "commit" phase of the prepare release section above.)
```

Comment:
for some reason, I always see files staged for commit at this point

Reply:
Ok, perhaps these are some files that are copied around by rsync.. It would be better if these where committed before you worked on the release notes.. Can you try to change the order? So run `deploy` command, then commit the remaining stuff (and merge it to master), then update the release and then publish it? The reason why I am saying this is that the release will create a tag - that tag now links to an older version of the master branch.. that's not ideal.

Reply:
I reordered this